### PR TITLE
PUB-1473 Care Standards List template Pdf

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1473

### Change description ###

Pdf template for Care Standards list.

Image 20-10-2022 at 12 47

Image 20-10-2022 at 12 47 (1)

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+        context.setVariable("publicationTime", DateHelper.formatTimeStampToBst(publicationDate, language,
+                                                                               true, false));
+        context.setVariable("contentDate", metadata.get("contentDate"));
+        context.setVariable("provenance", metadata.get("provenance"));
+        context.setVariable("email", artefact.get("venue").get("venueContact").get("venueEmail").asText());
+        context.setVariable("i18n", languageResources);
+
+        LocationHelper.formatCourtAddress(artefact, "\n", true);
+
+        context.setVariable("cases", TribunalNationalListsManipulation.processRawListData(artefact, language));
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java:34]: I don't believe you need the 'TribunalNationalListsManipulation' on this line, just the method name should do it

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java [hunk 2]
```diff
@@ -22,14 +45,14 @@ public static List<PrimaryHealthList> processRawListData(JsonNode data, Language
                                                                          language, false, false,
                                                                          "dd MMMM");
                     session.get("sittings").forEach(sitting -> {
-                        DateHelper.calculateDuration(sitting, language, true);
+                        DateHelper.calculateDuration(sitting, Language.ENGLISH, true);
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java:48]: Should this be kept as the language coming in from the metadata?

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java [hunk 3]
```diff
@@ -22,14 +45,14 @@ public static List<PrimaryHealthList> processRawListData(JsonNode data, Language
...
                         sitting.get("hearing").forEach(hearing -> {
                             String hearingType = hearing.get("hearingType").asText();
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java:50]: Hearing type can be null, so will need a null check around this

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java (hunk 1 above)
Comment (junaidiqbalmoj) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java]: fixed.

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java (hunk 2 above)
Comment (junaidiqbalmoj) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/TribunalNationalListsManipulation.java]: fixed.

**Review by ChrisS1512 — APPROVED** [review]

**Review by KianKwa — APPROVED** [review]

**Pull request merged**